.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://www.googleapis.com/youtube/v3/search?key=AIzaSyCR7rSc_jX6yUhB5z8GBV4B-nEzWQdgFlg&type=video&part=snippet&maxResults=10&q=
```

Comment:
Never commit keys or credentials in a code repository. Look for a way to remove these completely from your git history and find out how to provide them within your hosting platform, instead.